### ISSUE TITLE
Invalid Community Guide

### ISSUE THREAD

**Issue by sai-prasad-1** [meshery/meshery.io#1054]
### Description
in home page under the section that contains community meetings and community calendar links ,the link for community Guide is redirected to a google docs link that is showing 'Sorry, the file you have requested has been deleted.', also to note the url for the google docs contains the edit slug in  it "https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit"

### Expected Behavior
The  link should be opening a valid Google docs file

### Screenshots
![Screenshot from 2023-03-04 11-28-09](https://user-images.githubusercontent.com/90447924/222878857-d09ca99a-bc5e-44d0-91c8-20f1851c3c04.png)

### Enviroment:
 - Browser:   Firefox

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

**Comment by leecalcote** [comment]
@sudhanshutech fyi

**Comment by sudhanshutech** [comment]
Hey @sai-prasad-1 nice to point out, the doc was deleted mistakenly that's why it is redirecting to the wrong link.
If you want to work on this then redirect the link to  https://meshery.io/community/newcomers

**Comment by sai-prasad-1** [comment]
yes, i would like to work on the issue, and also @sudhanshutech  the "[Meshery Contributors' Welcome Guide] put out by the bot on this issue as an comment is also pointing to the wrong link

**Comment by sudhanshutech** [comment]
That's amazing @sai-prasad-1 , here you go

**Comment by sudhanshutech** [comment]
For the bot stuff, we need to create seperate issue
